css/desktop.css

```diff
@@ -0,0 +1,287 @@
+  }
+
+  .text-description {
+    font-size: 1.2vw;
+  }
+
+  .headline {
+    font-size: 2.2vw;
+    font-weight: 700;
+  }
+
+  hr {
+    /* margin: 0 auto; */
+    width: 3%;
+  }
+
+  .nav-ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 90%;
+  }
+
+  .nav-ul li {
+    display: flex;
+    justify-content: center;
+  }
+
+  .nav-ul a {
+    font-size: 1.1vw;
+    color: #868686;
+    font-weight: 800;
+  }
+
+  #webLogo {
+    display: flex;
+    width: 25%;
+    justify-content: flex-start;
+  }
+
+  #openNav {
+    display: none;
+  }
+
+  .register {
+    padding: 2% 3%;
+    border: 2px solid #ec5242;
+  }
+
+  /* INDEX HTML */
+  #home {
+    width: 66%;
+  }
+
+  #services {
+    padding-bottom: 5%;
+  }
+
+  #home .text-description {
+    font-size: 1vw;
+    border-color: #efefef;
+  }
+
+  #p_contact {
+    color: #7a7a7a;
+    font-size: 1.1vw;
+    font-weight: 700;
+  }
+
+  #detail_text small {
+    font-size: 1.1vw;
+    font-weight: 800;
+    text-decoration: underline;
+  }
+
+  .programs {
+    flex-direction: row;
+    padding: 4% 12%;
+  }
+
+  .program {
+    flex-direction: column;
+    padding: 1%;
+    gap: 4%;
+    width: 20%;
+    text-align: center;
+  }
+
+  .program img {
+    width: 30%;
+  }
+
+  .program img,
+  .program p {
+    margin: 0;
+  }
+
+  .program-name {
+    font-weight: 700;
+    font-size: 1.6vw;
+  }
+
+  .program-description {
+    font-weight: 700;
+    font-size: 0.8vw;
+    line-height: 189%;
+  }
+
+  .btn {
+    display: none;
+  }
+
+  #services a {
+    display: block;
+    text-decoration: underline;
+    color: #fff;
+    font-size: 1.3vw;
+    width: 100%;
+  }
+
+  #services .programs {
+    gap: 0.2em;
+  }
+
+  #services .program {
+    padding: 2% 1%;
+    gap: 0.5em;
+  }
+
+  /* FEATURES */
+  #features .programs {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  #features .program {
+    background-color: unset;
+    width: 100%;
+    flex-direction: row;
+    text-align: unset;
+  }
+
+  .speaker-head {
+    font-size: 1.6vw;
+    color: #000;
+    font-weight: 700;
+  }
+
+  .speaker-title {
+    font-size: 1.04vw;
+    color: #f78c89;
+    font-weight: 700;
+  }
+
+  .feature-hr {
+    width: 10%;
+  }
+
+  .speaker-description {
+    font-size: 1vw;
+    color: #000;
+    font-weight: 700;
+  }
+
+  .speaker-hidden {
+    display: flex;
+  }
+
+  /* LOGO SECTION */
+  #advertise {
+    padding: 3%;
+    display: flex;
+    justify-content: center;
+  }
+
+  #logoSection p {
+    font-size: 1.1vw;
+    margin: 0;
+    width: 50%;
+  }
+
+  #img_logo {
+    width: 35%;
+  }
+
+  /* ARCHIVES */
+  #past p {
+    padding: unset;
+  }
+
+  #archives {
+    padding-bottom: 4%;
+  }
+
+  #archiveList {
+    flex-direction: row;
+    justify-content: center;
+    gap: 1em;
+  }
+
+  .image-text {
+    font-size: 3.2vw;
+  }
+
+  .image-subtitle {
+    font-size: 1.4vw;
+  }
```

Comment:
- [ ] Great work implementing the red transparent background and image text as requested by the previous reviewer. The image texts overlap each other and make it difficult for the users to read, Kindly fix this. 👍 😎 

| First Header  | Second Header |
| -------------- | ----------------- |
| ![image](https://github.com/rubydevi/the-korner-capstone/assets/77106962/e0510ef9-92eb-4804-91ac-6842efd14e86)  | ![image](https://github.com/rubydevi/the-korner-capstone/assets/77106962/ccccf1f9-75c6-4de2-be62-39049e6e4f66)  |

Reply:
@Gambit142 Hey please check again. Sometimes it takes time to load right after I push from my terminal. Please respond after you receive this comment. I had to make sure since I don't see it from my side.

Reply:
Hi @rubydevi, I am sorry for the late response. I reloaded it several times and it gave the same problem. 👍